src/ui/popup/App.tsx

```diff
@@ -300,6 +300,8 @@ class App extends React.Component<PopupAppComponentProps, InitialState> {
                 data-disabled="true"
                 role="menu"
                 style={{
+                  borderBottomLeftRadius: 0,
+                  borderTopLeftRadius: 0,
```

Comment:
Found a better way - just remove the div "dropdown" parent, and add 
``` 
borderLeftColor: 'rgb(176, 132, 0)',
``` 
to the style (removing the two radius lines you added) and maybe change '-3px' to '-2px' for somewhat uniform spacing in translate3d.

See if that works better for you.